### DISCUSSION TITLE
Compute specificity of binders to the target?

### DISCUSSION THREAD

**Discussion by remybonnav** [RosettaCommons/RFdiffusion#50] [General]
Is it possible to implement a step where when we design binders that are specific to a protein and that do not bind to other list of protein?

**Comment by roccomoretti** [comment]
I do not believe that RFDiffusion has any explicit facilities for negative design. Keeping in mind that RFDiffusion proper is only concerned with the backbone structure, my likely suggestion on how to proceed would be to take a look at the similar interfaces on each entry in the set, and see if there's large shape differences between them. If so, you could potentially work with a "bump and hole" type approach, where you tell RFDiffusion about your desire to have a backbone shaped appropriately to match the bump-and-hole features which distinguish the two.

If the difference between the interfaces is more energetic (hydrophobic/polar/electrostatic/hbond) versus steric, then I think RFDiffusion will have difficulties in explicitly designing to the difference, as it's primarily backbone focused rather than sequence/sidechain focused. You may be able to pick up discrimination on the backend design process (e.g. ProteinMPNN), though. I don't recall ProteinMPNN having explicit negative design capabilities either, but you might be able to put something together with ProteinMPNN following a RECON ([1](https://pubmed.ncbi.nlm.nih.gov/26147100/)) ([2](https://pubmed.ncbi.nlm.nih.gov/30642961/)) like approach. RECON doesn't do negative design either, but the central idea behind it is an iterative design process with adjusted constraints at each step. You could do something similar by running a naive design against all the targets, then telling ProteinMPNN to downweight those residue identities which are promiscuous binders.

If you do want explicit negative design, then Rosetta's MPI-MSD ([3](https://pubmed.ncbi.nlm.nih.gov/21754981/)) ([4](https://www.rosettacommons.org/docs/latest/application_documentation/design/mpi-msd)) might be a possibility. That uses Rosetta's packer to do design, though, rather than ProteinMPNN.

**Reply by remybonnav** [reply]
Great, thanks a lot for this really detailed answer.
